openslides/users/serializers.py

```diff
@@ -45,6 +49,75 @@ class Meta:
             'is_active',)
 
 
+class UserCreateUpdateSerializer(ModelSerializer):
+    """
+    Serializer for users.models.User objects.
+
+    Serializes data to create new users or update users.
+
+    Do not use this for list or retrieve requests.
+    """
+    groups = PrimaryKeyRelatedField(
+        many=True,
+        queryset=Group.objects.exclude(pk__in=(1, 2)),
+        help_text=ugettext_lazy('The groups this user belongs to. A user will '
+                                'get all permissions granted to each of '
+                                'his/her group.'))
```

Comment:
Done. But this text was copied from django.contrib.auth.models.Permission.groups.